cgmanifest.json

```diff
@@ -5040,8 +5040,8 @@
         "type": "other",
         "other": {
           "name": "guile",
-          "version": "2.0.14",
-          "downloadUrl": "ftp://ftp.gnu.org/pub/gnu/guile/guile-2.0.14.tar.gz"
+          "version": "3.0.9",
+          "downloadUrl": "ftp://ftp.gnu.org/pub/gnu/guile/guile-3.0.9.tar.gz"
```

Comment:
it looks like an https link is available, can you please make the switch? `https://ftp.gnu.org/gnu/guile/guile-3.0.9.tar.gz`

Reply:
(from https://www.gnu.org/software/guile/download/#releases)

Reply:
how do i replace an existing uploaded tar.gz?  i think the [Publish Source Tarball pipeline fails when you use the same name?  ( i mean, it is probably not neccessary, since the tarball should be the same ... but am curious nonetheless)